kustomize/base/shard/resources/deploy.yaml

```diff
@@ -9,10 +9,10 @@ spec:
   replicas: 1
   template:
     spec:
-      serviceAccountName: shard-operator
+      serviceAccountName: shard-operator-knative
       containers:
         - image: event-bridge-shard-operator
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
Why? I think we should stay with `IfNotPresent`

Reply:
because it is a workaround for testing. will be removed